plugins/workspace-minimap/src/minimap.ts

```diff
@@ -55,7 +55,11 @@ export class Minimap {
     init(): void {
       this.minimapWorkspace.scrollbar.setContainerVisible(false);
       this.primaryWorkspace.addChangeListener((e) => void this.mirror(e));
-      window.addEventListener('resize', () => {
+      Blockly.browserEvents.bind(this.minimapWorkspace.svgGroup_,
+          'mousedown', this, this.onClick);
+      Blockly.browserEvents.bind(this.minimapWorkspace.svgGroup_,
+          'mousemove', this, this.onMouseMove);
+      window.addEventListener('resize', (e) => {
```

Comment:
If a parameter is unused, I think you can also use `_` as a placeholder name! Not important though, just thought it might be interesting :P